osu.Game/Overlays/Profile/Sections/Ranks/DrawableProfileScore.cs

```diff
@@ -213,11 +213,36 @@ private void load(RulesetStore rulesets)
 
         private Drawable createDrawablePerformance()
         {
-            if (!Score.PP.HasValue)
+            if (Score.PP.HasValue)
             {
-                if (Score.Beatmap?.Status.GrantsPerformancePoints() == true)
-                    return new UnprocessedPerformancePointsPlaceholder { Size = new Vector2(16), Colour = colourProvider.Highlight1 };
+                return new FillFlowContainer
+                {
+                    AutoSizeAxes = Axes.Both,
+                    Direction = FillDirection.Horizontal,
+                    Children = new[]
+                    {
+                        new OsuSpriteText
+                        {
+                            Anchor = Anchor.BottomLeft,
+                            Origin = Anchor.BottomLeft,
+                            Font = OsuFont.GetFont(weight: FontWeight.Bold),
+                            Text = $"{Score.PP:0}",
+                            Colour = colourProvider.Highlight1
+                        },
+                        new OsuSpriteText
+                        {
+                            Anchor = Anchor.BottomLeft,
+                            Origin = Anchor.BottomLeft,
+                            Font = OsuFont.GetFont(size: 12, weight: FontWeight.Bold),
+                            Text = "pp",
+                            Colour = colourProvider.Light3
+                        }
+                    }
+                };
+            }
 
+            if (Score.Beatmap?.Status.GrantsPerformancePoints() != true)
```

Comment:
Is there a reason this can't just be grouped in with the `UnrankedPerformanePointsPlaceholder` below? It seems like basically the same case.

Reply:
Well if we're being strict then t's not the same case. This handles stuff like scores on loved/graved beatmaps. Web uses a different tooltip for those ("Only ranked beatmaps award pp").

Reply:
The duplication of the symbol looks really awful, I'll alter the code slightly.

Reply:
not sure that is necessary.

Reply:
Might as well if it looks weird to someone already.

Reply:
Possibly. Relevant source:

https://github.com/ppy/osu-web/blob/64d540df6e8296e5090a9cc56a59189058585b67/resources/js/utils/beatmap-helper.ts#L114-L116
https://github.com/ppy/osu-web/blob/64d540df6e8296e5090a9cc56a59189058585b67/resources/js/profile-page/play-detail.tsx#L125

Reply:
Again, if we're being pedantic, then not quite because of https://github.com/ppy/osu-web/blob/64d540df6e8296e5090a9cc56a59189058585b67/resources/js/profile-page/play-detail.tsx#L126-L128

tl;dr loved maps should show a fontawesome heart instead of a dash.